com.unity.render-pipelines.core/Runtime/Debugging/Prefabs/Scripts/DebugUIHandlerEnumField.cs

```diff
@@ -30,6 +31,9 @@ internal override void SetWidget(DebugUI.Widget widget)
         /// <returns>State of the widget.</returns>
         public override bool OnSelection(bool fromNext, DebugUIHandlerWidget previous)
         {
+            // TODO: Put these back once we update the prefab to have buttons in it...
```

Comment:
What is status on these?

Reply:
Good catch - I can comment the code back in now.